SignalServiceKit/tests/Messages/OWSMessageSenderTest.m

```diff
@@ -221,16 +261,14 @@ - (void)setUp
     // Successful Sending
     TSNetworkManager *successfulNetworkManager = [[OWSMessageSenderFakeNetworkManager alloc] initWithSuccess:YES];
     self.successfulMessageSender = [[OWSMessageSender alloc] initWithNetworkManager:successfulNetworkManager
```

Comment:
Is this right? IIRC you removed all the dependencies from the message sender except the `primaryStorage`?

Reply:
This is one of the `BROKEN_TESTS`. It's coming.